snippets/dart_snippets/lib/send_payment.dart

```diff
@@ -5,14 +5,14 @@ Future<SendPaymentResponse> sendPayment({required String bolt11}) async {
   // ANCHOR: send-payment
   // Set the BOLT11 invoice you wish to pay
   PrepareSendResponse prepareSendResponse = await breezSDKLiquid.instance!.prepareSendPayment(
-    req: PrepareSendRequest(invoice: "..."),
+    req: PrepareSendRequest(destination: "Invoice, Liquid BIP21 or address", amountSat: 2000 as BigInt),
```

Comment:
I think it's good in the docs to show the parameter is optional. e.g. my the naming of the variable
```
BigInt optionalAmountSat = BigInt.from(2000);
```